3p/ampcontext.js

```diff
@@ -41,30 +41,20 @@ export class AmpContext extends IframeMessagingClient {
   constructor(win) {
     super(win);
     this.setupMetadata_();
+    this.getHostWindow();
```

Comment:
We can call `#getHostWindow` [there](https://github.com/ampproject/amphtml/pull/6441/files#diff-a65cdac49d5347e8813ad2ebfae7d347R77), though.